buildkite/job.go

```diff
@@ -121,29 +120,35 @@ func (j *Job) Run(agent *Agent) error {
 		envSlice = append(envSlice, fmt.Sprintf("%s=%s", key, value))
 	}
 
-	// Mark the build as started
-	j.StartedAt = time.Now().UTC().Format(time.RFC3339Nano)
-	_, err := agent.Client.JobUpdate(j)
-	if err != nil {
-		// We don't care if the HTTP request failed here. We hope that it
-		// starts working during the actual build.
-	}
+	// The HTTP request we'll be sending it to
+	logStreamerRequest := agent.Client.GetSession().NewRequest("POST", "jobs/"+j.ID+"/log")
+
+	// Chunks will keep on retrying until they upload
```

Comment:
Not sure what logic to choose here. I can say: "the chunk will forever keep trying" or "if it doesn't upload within 5 minutes of retrying, give up".